resources/assets/js/store/modules/page.js

```diff
@@ -26,7 +26,8 @@ const state = {
 	scale: .4,
 	loaded: false,
 	dragging: false,
-	currentSavedState: ''
+	currentSavedState: '',
+	isPublished: true
```

Comment:
What is this isPublished?

Reply:
@sy238-unikent take a look at the latest version for comments